packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -212,8 +211,8 @@ export function resolveStyleRules(
   styles: MakeStylesStyle,
   unstable_cssPriority: number = 0,
 ): [CSSClassesMap, CSSRulesByBucket] {
-  // expandShorthand() and resolveProxyValues() are recursive functions and should be evaluated once for a style object
-  const expandedStyles: MakeStylesStyle = expandShorthand(resolveProxyValues(styles));
+  // resolveProxyValues() is recursive function and should be evaluated once for a style object
+  const expandedStyles: MakeStylesStyle = resolveProxyValues(styles);
 
   return resolveStyleRulesInner(expandedStyles, unstable_cssPriority);
```

Comment:
Should this be renamed to something like

```suggestion
  const proxyStyles: MakeStylesStyle = resolveProxyValues(styles);
  
  return resolveStyleRulesInner(proxyStyles, unstable_cssPriority);
```